subjects/animals/README.md

```diff
@@ -0,0 +1,65 @@
+## Gatecrashers
+
+### Instructions
+
+Modern JavaScript implements high level tools for creating and managing classes, still most of those tools rely on low level functions still present and in use in some libraries and legacy code.
+
+In order to understand how `class` and `inheritance` works you will have to recreate a similar behavior by using only the low level functions provided by the language.
+
+You will have `Animal` which will be the equivalent of a base class and will have the following fields:
+- `canEat`, `canBreath`, `isAlive`: All booleans, set to true.
+- `WhoAmI`: Function that return a string, here it will be `"I'm an animal"`.
+
+The following objects will inherit from `Animal` and add/override fields as follow:
+- `Dog`: add field `canRun` set to `true`.
+- `Bird`: add field `canFly` and `makeEggs` set to `true`.
+- `Dodo`: override `canFly` and `isAlive` to `false`.
```

Comment:
Also, there might be some confusion because by `overriding` it could be understood to modify the value in the prototype, instead of creating its own properties.

Reply:
done for the grammar part.
About the confusion, overriding a property is what I've always saw to be in use in those cases, if you have a better term let me know and I'll check for it, but as of now I think it's clear enough what the intended behavior should be

Reply:
The confusion comes from the fact that overriding is ambiguous. It's not wrong, just ambiguous when describing what to do. There is more than one way of overriding a property. For instance [`prototype` overriding example](https://www.c-sharpcorner.com/blogs/method-overriding-in-javascript)

I can guess by context that you mean to `add` new properties to the Dodo object with the new value set to `false`. But by reading `override` what I understand is that what needs to be done is modifying the prototype in `Bird` to `false`. Override could be either to make the inheriting object have it's own value, or modifying the prototype.

An easy fix if you still wish to use the term `override`, which is not wrong, is to add after `false` the following: `without modifying the prototype`.

It would be nice to add an example where it is required to override the prototype as well. It could be at the end to be able to test this exercise first.

Storyline suggestion: Due to an atmosphere thinning event, some heavy birds can't fly any more. ([scientific base](https://www.scientificamerican.com/article/early-earth-s-atmosphere-was-surprisingly-thin/)). You could use your own creative way of testing this concept, not necessarily with birds, or this suggested storyline, it could be with fish, and due to global warming (some of) them becoming extinct, or any other thing you think about that makes some sense. :slightly_smiling_face:

Reply:
Overriding the prototype doesn't seems like a good idea, we've discussed it and it's true that it may add some value to the student in terms of knowledge, but we need to rethink the exercise a bit in order for the override to make sense. We'll discuss it further to see how and if to implement this new required task

Reply:
Just a note to remember that we should warn that it is for learning purposes, since it's a building block of how the language works, and perhaps add a link with the reasons why it's generally considered a bad practice to override the prototype as we've discussed. :+1: